packages/x-adapter/src/mappers/__tests__/combine-mappers.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { DeepPartial } from '@empathyco/x-utils';
+import { Mapper } from '../../types/mapper.types';
+import { combineMappers } from '../combine-mappers';
+
+describe('combineMappers tests', () => {
```

Comment:
`'combineMappers util'` we already know this is a test

Reply:
Sure, I've changed it!